pkg/util/trello/trello.go

```diff
@@ -44,27 +50,41 @@ func (c *Client) CreateList(board *trello.Board, listName string) (*trello.List,
 	return c.Client.CreateList(board, listName, trello.Defaults())
 }
 
-func (c *Client) GetBoardIdAndListId() (map[string]interface{}, error) {
-	res := make(map[string]interface{})
+// GetBoardIdAndListId get the board, which board name == kanban, and board desc == owner/repo
+func (c *Client) GetBoardIdAndListId(owner, repo, kanban string) (map[string]interface{}, error) {
 
+	res := make(map[string]interface{})
 	bs, err := c.Client.GetMyBoards()
 	if err != nil {
 		return nil, err
 	}
 
 	for _, b := range bs {
-		lists, err := b.GetLists()
-		if err != nil {
-			return nil, err
+		if checkTargetBoard(owner, repo, kanban, b) {
+			lists, err := b.GetLists()
+			if err != nil {
+				return nil, err
+			}
+			if len(lists) != DefaultListsNumber {
+				log.Errorf("Unknown lists format: len==%d", len(lists))
+				return nil, fmt.Errorf("unknown lists format: len==%d", len(lists))
+			}
+			res["boardId"] = b.ID
+			res["todoListId"] = lists[0].ID
+			res["doingListId"] = lists[1].ID
+			res["doneListId"] = lists[2].ID
 		}
-		if len(lists) != 3 {
-			log.Errorf("Unknown lists format: len==%d.", len(lists))
-			return nil, fmt.Errorf("unknown lists format: len==%d", len(lists))
-		}
-		res["boardId"] = b.ID
-		res["todoListId"] = lists[0].ID
-		res["doingListId"] = lists[1].ID
-		res["doneListId"] = lists[2].ID
 	}
 	return res, nil
 }
+
+func checkTargetBoard(owner, repo, kanban string, b *trello.Board) bool {
+	if b.Name == kanban && b.Desc == boardDesc(owner, repo) {
+		return true
+	}
+	return false
+}
+
+func boardDesc(owner, repo string) string {
+	return fmt.Sprintf("Description managed by DevStream,don't change.%s/%s", owner, repo)
```

Comment:
"Description is managed by DevStream, please don't modify. %s/%s"

or simply

"Managed by DevStream, please don't modify. %s/%s"

- There should be a space after punctuation.
- change -> modify (to be consistent with the readme)
- add "please"